web/src/beta/features/AccountAndWorkSpaceSetting/innerPages/AccountSetting/index.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+  Collapse,
+  TextInput,
+  Typography,
+  IconButton
+} from "@reearth/beta/lib/reearth-ui";
+import { InputField } from "@reearth/beta/ui/fields";
+import { PasswordPolicy } from "@reearth/services/config/passwordPolicy";
+import { useT } from "@reearth/services/i18n";
+import { styled } from "@reearth/services/theme";
+import { FC, useState } from "react";
+
+import { UpdatePasswordType } from "../../hooks";
+import { InnerPage, SettingsWrapper, SettingsFields } from "../common";
+
+import PasswordModal from "./PasswordModal";
+
+type Props = {
+  informationData: { name?: string; email?: string };
+  passwordPolicy?: PasswordPolicy;
+  onUpdateUserPassword: ({
+    password,
+    passwordConfirmation
+  }: UpdatePasswordType) => Promise<void>;
+};
+
+const AccountSetting: FC<Props> = ({
+  passwordPolicy,
+  onUpdateUserPassword,
+  informationData
+}) => {
+  const t = useT();
+  const [passwordModalClosed, setPasswordModalClosed] = useState<boolean>(true);
```

Comment:
```suggestion
  const [changePasswordModal, setChangePasswordModal] = useState<boolean>(false);
```
can you rename to match its functionality

Reply:
Change name to changePasswordModal. It's more simple to use. Please check.
https://github.com/reearth/reearth-visualizer/pull/1155/commits/528ac026693a7a96c8a8a43b11b70df4cdfe1c8c